### PR TITLE
fix: set max request body size to 2MB on API

### PR DESCRIPTION
**Pull request by leite08**
Ref. metriport/metriport-internal#334

### Dependencies

- none

### Description

Set max request body size to 2MB on API.

### Release Plan

- asap

### CODE REVIEW

File: api/app/src/app.ts [hunk 1]
```diff
@@ -10,8 +10,8 @@ import mountRoutes from "./routes/index";
 
 const app: Application = express();
 app.use(helmet()); // needs to come before any route declaration, including cors()
-app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
+app.use(express.json({ limit: 2_000 }));
```
Comment (Goncharo) [inline api/app/src/app.ts:13]: As long as we're sure this is kb

**Review by Goncharo — APPROVED** [review]
🚢

**Review by Orta21 — APPROVED** [review]

**Review by Goncharo — CHANGES_REQUESTED** [review]
looking at https://expressjs.com/en/api.html
 
"If this is a number, then the value specifies the number of bytes"

should this be `"2mb"`?

File: api/app/src/app.ts (hunk 1 above)
Comment (leite08) [reply api/app/src/app.ts:13]: No, my bad, this param is in bytes, so should be 2_000_000... I'll just use "2mb"

**Comment by leite08** [comment]
Updated  to "2mb", good catch 👍

**Comment by leite08** [comment]
Also tested local with Postman, sending a payload of 17kb with the previous version (`2_000`) it fails... with the new one (`"2mb"`) it works.

**Review by Goncharo — APPROVED** [review]
🚢

**Pull request merged**